microsoft-edge/web-platform/translator-api.md

```diff
@@ -0,0 +1,429 @@
+
+```javascript
+// Create a Translator session.
+const translatorSession = await Translator.create({
+  sourceLanguage: "en",
+  targetLanguage: "es"
+});
+
+// Create an AbortController object.
+const abortController = new AbortController();
+
+// Use the model session to translate some text and pass the AbortController
+// object by using the signal option.
+const stream = translatorSession.translateStreaming(sourceText, {
+    signal: abortController.signal
+});
+
+// Later, perhaps when the user presses a "Stop" button, call the abort()
+// method on the AbortController object to stop translating text.
+abortController.abort();
+```
+
+
+<!-- ====================================================================== -->
+## Destroy a session (`destroy()`)
+
+After translating text, destroy the session to let the browser know that you don't need the language model anymore, so that the model can be unloaded from memory.
+
+You can destroy a session in two different ways:
+* By using the `destroy()` method.
+* By using an `AbortController`.
+
+Details are below.
+
+
+<!-- ------------------------------ -->
+#### Destroy a session by calling `destroy()`
+
+To destroy a session by calling `destroy()` with a `Translator` session:
+
+```javascript
+const session = await Translator.create({
+  sourceLanguage: "en",
+  targetLanguage: "es"
+});
+
+// Later, destroy the session by using the destroy method.
+session.destroy();
+```
+
+
+<!-- ------------------------------ -->
+#### Destroy a session by using `AbortController`
+
+To destroy a session by creating an `AbortController` object, create a `Translator` session, and then call `abort()`:
+
+```javascript
+// Create an AbortController object.
+const controller = new AbortController();
+
+// Create a Translator session and pass the 
+// AbortController object by using the signal option.
+const session = await Translator.create({
+  sourceLanguage: "en",
+  targetLanguage: "es",
+  signal: controller.signal
+});
+
+// Later, perhaps when the user interacts with the UI, destroy the session by
+// calling the abort() function of the AbortController object.
+controller.abort();
+```
+
+
+<!-- ====================================================================== -->
+## Send feedback
+
+We're very interested in learning about the range of scenarios for which you intend to use the Translator API, any issues with the API or expert model, and whether other task-specific, built-in APIs would be useful.
+
+To send feedback about your scenarios and the tasks you want to achieve, please add a comment to [the Translator API feedback issue](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/XYZ)<!-- todo: create a new feedback issue url/template -->, or [Report an issue about the Translator API in Microsoft Edge](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/XYZ)<!-- todo: create a new feedback issue url/template; https://github.com/MicrosoftEdge/MSEdgeExplainers/pull/1158 -->.
```

Comment:
Correctly link to template result of PR 1158 https://github.com/MicrosoftEdge/MSEdgeExplainers/pull/1158/files

Check how done in similar articles:
* https://review.learn.microsoft.com/microsoft-edge/web-platform/prompt-api?branch=pr-en-us-3566#send-feedback
* https://review.learn.microsoft.com/microsoft-edge/web-platform/writing-assistance-apis?branch=pr-en-us-3566#send-feedback